docs/tutorial/first-app.md

```diff
@@ -104,6 +113,14 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteRule . index.php [L]
 ```
 
+nginx does not use `.htaccess` files, so you will need to add the following to your server configuration in the `location` block:
+
+```
+if (!-e $request_filename){
+    rewrite ^(.*)$ /index.php break;
```

Comment:
Well I suppose if people are changing the entry point, we could assume they know to change the nginx config :)